packages/shared/src/components/modals/UpvotedPopupModal.spec.tsx

```diff
@@ -38,7 +38,7 @@ const upvotedCommentQuery: UpvotedPopupModalProps = {
   requestQuery: {
     queryKey: ['commentUpvotes', commentId],
     query: COMMENT_UPVOTES_BY_ID_QUERY,
-    params: { id: commentId, first: DEFAULT_UPVOTES_PER_PAGE },
+    params: { id: commentId, first: DEFAULT_UPVOTES_PER_PAGE, after: '' },
```

Comment:
Do we send `after` always like that, just looks like something that can break if we don't do it on app side?

Reply:
Or is this just a product of `initialPageParam`?

Reply:
Yes result of `initialPageParam` indeed it will send empty string instead of not sending it.